multiwallet_test.go

```diff
@@ -0,0 +1,35 @@
+package dcrlibwallet_test
+
+import (
+	"os"
+	"testing"
+	"testing/quick"
+
+	. "github.com/raedahgroup/dcrlibwallet"
+)
+
+// canUseDir returns true if the program can create the directory
+// if it doesn't exist or the directory is empty if it exists.
+// If will delete the directory if it is empty.
+func canUseDir(directory string) bool {
+	if os.MkdirAll(directory, os.ModePerm) == nil {
+		return os.Remove(directory) == nil
```

Comment:
Considering that this function is used to test the `NewMultiwallet` function, this should return nil. `NewMultiWallet` only attempts to create the directory if it does not exist, it does not remove any existing content from the directory - which is important since the root dir may contain a `wallet.db` file that will/should be linked to the new multiwallet instance, AFTER, the instance is created.

Reply:
Yeah, I sure knew that since it was documented somewhere...

Reply:
And the thing is that this removes the directory if it exists but is empty (like the code comment said) so all values for which `canUseDir` returns `true` are valid paths that do not exist.

Reply:
Ah I see. I guess that's the point I'm trying to make. A path may also be valid if it exists and is not empty. This function would return an error for any valid path that is not empty.

Reply:
It doesn't return an error...